projects/slipshow/programs/slipshow/tests/shell.nix

```
@@ -0,0 +1,41 @@
+{
+  sources,
+  ...
+}:
+{
+  name = "slipshow presentation";
+
+  nodes = {
+    machine =
+      { pkgs, ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.slipshow
+          sources.examples.slipshow.demo-shell
```

Actually, there is only 1 example, which is the basic one:

```suggestion
          sources.examples.slipshow.basic
```